tests/integration/actions/config/base.py

```diff
@@ -74,11 +77,21 @@ def test(self, request, tmux_session, step):
 
         if step.mask:
             # mask out some configuration that is subject to change each run
-            mask = "X" * 50
-            maskables = ["BECOME_PLUGIN_PATH", "CACHE_PLUGIN_CONNECTION", "COLLECTIONS_PATHS"]
-            for idx, line in enumerate(received_output):
-                if any(m in line for m in maskables):
-                    received_output[idx] = mask
+            maskables = [
+                "BECOME_PLUGIN_PATH",
+                "CACHE_PLUGIN_CONNECTION",
+                "COLLECTIONS_PATHS",
+                "DEFAULT_CALLBACK_PLUGIN_PATH",
+                "DEFAULT_LOCAL_TMP",
+            ]
+            # Determine if a menu if showing
+            match = re.search("CURRENT VALUE", received_output[0])
+            if match:
+                mask_start, mask_end = match.span()
+                mask = (mask_end - mask_start) * "X"
+                for idx, line in enumerate(received_output):
+                    if any(f"│{m}" in line for m in maskables):
+                        received_output[idx] = received_output[idx][0:mask_start] + mask
```

Comment:
I've taken a stub at dropping `re` and it seems cleaner with
```suggestion
            def _line_needs_masking(line):
                return any(f"│{m}" in line for m in maskables)

            mask_marker = "CURRENT VALUE"
            mask = len(mask_marker) * "X"

            try:
                mask_start = received_output[0].index(mask_marker)
            except ValueError:
                mask_start = None

            if mask_start is not None:
                received_output = [
                    line[:mask_start] + mask if _line_needs_masking(line)
                    else line
                    for line in received_output
                ]
```

Reply:
@cidrblock the only thing that is unclear is why the replaced thing is not just the masked value but the rest of the line too?

Reply:
The length of the value to mask can change, so I used the minimum possible column width which is always the length of the column header.  If we masked the the actual of the value with X * length of the value, we might still changes between developer system where the fixtures were generated.

Reply:
Ah, okay. Although, the second question would be — why not set a fixed TTY size in tests so that it would not depend on the developer's setup?

Reply:
we do, but that wouldn't cover a case where the value didn't go to the far right side of the screen.

Reply:
Interesting... FWIW I'm resolving this now.